docs/meetings/team/2023_01_23.md

```diff
@@ -0,0 +1,77 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:00
+
+**End Time** 10:36
+
+**Location** Discord
+
+**Date** January 23, 2023
+
+## 👋 Attendence
+
+1. Veronica
+2. Logan
+3. Edouard
+4. Matt
+
+***Attendence Comments***
+
+- Logan will be away next week for medical reasons
+
+## 🃏 Joke of the Day
+
+What do you call 8 hobbits? A hobbyte
+
+## 📢 Announcements
+
+- Week 3 GitStats due this week on Friday
+- Week 4 has a new deliverable we should ask the prof about
+- Rhubarb developer replied! Check Discord for a screenshot of the message
+
+## 📅 Last Week (Restrospective)
+
+- Thank you, Matt, for submitting GitStats for us!
+- Ed was working on making a formal API definition, but it's incomplete
+- We found that Postman was crashing (screen goes white when trying to preview, but the download works)
+- Veronica implemented blinking, but could implement types to improve it for TypeScript
+
+## 📋 This Week
+
+### Assets
+
+- Official assets with mouths and eyes are exported from Photoshop! Thank you, Logan!
+- If we want to have the arms move...
+  - Need to check with Client if we want breathing vs. poses because poses will require more from the artist
+  - Barb's arms are movable
+  - The boy's arms are attached to the body, which are attached to the feet and are NOT movable
+
+### Issues this week
+
+- Continue working on a formal API definition (Ed)
+- Figure out why Postman is crashing (Ed & Matt)
+  - This could have to do with ffmpeg scaling (Logan)
+  - Matt had it temporarily working after a re-install, but it's broken again
+- Implement more types and clean up current types (Veronica & Ed)
+- Rhubarb developer to-do's:
+  - Extended mouth shapes being used? double check (Veronica)
+  - Check command-line program's exit code rather than its output (Matt & Veronica)
+- Check Boy in Illustrator because his arms might not be movable due to flattening of layers (Veronica)
+- Need to find better test files with harder English words so that we can better showcase what we have (Everyone)
+
+## 🔮 Future Issues (after this week)
+
+- Export the assets in different arm positions after discussing at client meeting
+- If doing poses, will we trigger them randomly? How do we want to do this? Something to think about this week.
+
+## 🤝 Client Meeting
+
+- Showcase videos from both recognizers so that we can ask if they want quality vs. speed for English
+- We should also ask if they want breathing or specific poses (which would require more from the artist)
```

Comment:
Breathing wouldn't necessarily require extra work! We should just ask them what poses they think would be nice to have, and then see if their animator is willing to provide those. I think it's maybe good to check in with nazim and see how the artist is feeling about everything in regards to workload and our requests and such

Reply:
For breathing, we would probably need to show it in the poses, too. So that's why I'm a little concerned about showing the breathing because it would be required for each pose which could get complicated.

Definitely worth asking Nazim about the artist and the workload!

Reply:
Im thinking arm flapping = breathing for this case, and then for specific poses we can just stop the breathing animation and transition to the pose - kinda like how in video games the character has like an idle animation where they sorta jiggle up n down

Reply:
https://www.youtube.com/watch?v=cRoQ5gz1UwM similar to how this dude kinda like heaves, we could show it by moving the arms sorta in n out of the character I think - not sure tho could look bad